uxarray/grid/neighbors.py

```diff
@@ -0,0 +1,220 @@
+
+        xy = _prepare_xy_for_query(xy, use_radians)
+
+        # perform query with distance
+        if return_distance:
+            d, ind = self.tree.query(xy, k, return_distance, dualtree,
+                                     breadth_first, sort_results)
+
+            ind = np.asarray(ind, dtype=INT_DTYPE)
+
+            # only one pair was queried
+            if xy.shape[0] == 1:
+                ind = ind.squeeze()
+                d = d.squeeze()
+
+            d = np.rad2deg(d)
+
+            return d, ind
+
+        # perform query without distance
+        else:
+            ind = self.tree.query(xy, k, return_distance, dualtree,
+                                  breadth_first, sort_results)
+
+            ind = np.asarray(ind, dtype=INT_DTYPE)
+
+            if xy.shape[0] == 1:
+                ind = ind.squeeze()
+
+            return ind
+
+    def query_radius(self,
+                     xy: Union[np.ndarray, list, tuple],
+                     r: Optional[int] = 1.0,
+                     use_radians: Optional[bool] = False,
+                     return_distance: Optional[bool] = True,
+                     count_only: Optional[bool] = False,
+                     sort_results: Optional[bool] = False):
+        """Queries the tree for neighbors within a radius ``r``.
+
+        Parameters
+        ----------
+        XY : array_like
+           coordinate pairs in degrees (lon, lat) to query
+        r: distance in degrees within which neighbors are returned
+            r can be a single value , or an array of values of shape x.shape[:-1] if different radii are desired for each point.
+        use_radians : bool, default=True
+            if True, queries assuming xy are inputted in radians, not degrees
+        return_distance : bool, default=False
+            if True, return distances to neighbors of each point if False, return only neighbors Note that unlike the query() method, setting return_distance=True here adds to the computation time. Not all distances need to be calculated explicitly for return_distance=False. Results are not sorted by default: see sort_results keyword.
+        count_only : bool, default=False
+            if True, return only the count of points within distance r if False, return the indices of all points within distance r If return_distance==True, setting count_only=True will result in an error.
+        sort_results : bool, default=False
+           if True, the distances and indices will be sorted before being returned. If False, the results will not be sorted. If return_distance == False, setting sort_results = True will result in an error.
+
+
+        Returns
+        -------
+        d : ndarray of shape (xy.shape[0], k), dtype=double
+            Each entry gives the list of distances to the neighbors of the corresponding point.
+        ind : ndarray of shape (xy.shape[0], k), dtype=INT_DTYPE
+            Each entry gives the list of indices of neighbors of the corresponding point.
```

Comment:
Similar to the above comments; please consider rephrasing these two descriptions to make them easy to read